sdks/python/apache_beam/examples/complete/juliaset/setup.py

```diff
@@ -51,7 +51,7 @@ class build(_build):  # pylint: disable=invalid-name
 # two commands will have to be added:
 #
 #     ['apt-get', 'update'],
-#     ['apt-get', '--assume-yes', install', 'libjpeg62'],
+#     ['apt-get', '--assume-yes', 'install', 'libjpeg62'],
 #
 # First, note that there is no need to use the sudo command because the setup
 # script runs with appropriate access.
```

Comment:
<!--thread_id:cc_152665040_t; commit:7fe7732bbf7721231ef7ec072307e533f006aef3; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Note: this line is Dataflow-specific until the FnAPI / containerization effort lands.

<!--section:body-->
Acknowledged. This needs some additional clean up.